build.sbt

```diff
@@ -399,7 +399,7 @@ lazy val metals = project
       "org.jboss.xnio" % "xnio-nio" % "3.8.9.Final",
       // for persistent data like "dismissed notification"
       "org.flywaydb" % "flyway-core" % "9.20.1",
-      "com.h2database" % "h2" % "2.2.220",
+      "com.h2database" % "h2" % "2.1.114",
```

Comment:
Used to be "2.1.214".

Reply:
I should have just done a real revert :/

What is the logic behind the versions here even